apps/server/src/modules/user-migration/mapper/page-content.mapper.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common';
+import { PageContentDto } from '../service/dto/page-content.dto';
+import { PageContentResponse } from '../controller/dto/page-content.response';
+
+@Injectable()
+export class PageContentMapper {
```

Comment:
I don't know where the information came from that mappers should be injectable, but almost everywhere in the project these are static methods

Reply:
We talked about this and that there are different opinions on this. This is added to ARC Topic list to have a definitive answer solved in the weekly.